index.js

```diff
@@ -37,19 +39,21 @@ class ClinicFlame extends events.EventEmitter {
       path: this.path,
       identifier: '{pid}' // replaced with actual pid by 0x
     })
-
-    callbackify(x({
-      argv,
-      onPort: this.detectPort ? onPort : undefined,
-      onProcessExit: () => {
-        this.emit('analysing')
-      },
-      pathToNodeBinary: args[0],
-      collectOnly: true,
-      writeTicks: true,
-      outputDir: paths['/0x-data/'],
-      workingDir: '.' // 0x temporary working files, doesn't support placeholders like {pid}
-    }), done)
+    // allow time delay to be specified before we start collecting data
+    setTimeout(() => {
+      callbackify(x({
```

Comment:
Since 0x is the one that's actually starting our subprocess and profiling, wrapping the entire thing in a delay just means we're starting the subprocess later. What we really want is a bit more complicated (hopefully just a bit … :grimacing: ). The subprocess should start immediately, but we should discard any of the data that is collected before the timeout.